src/server.jsx

```diff
@@ -52,7 +52,7 @@ export default locals => {
           <meta property="og:type" content="website" />
           <meta property="og:title" content={ title } />
           <meta property="og:description" name="description" content={ description } />
-          <meta property="og:image" content={ Logo } />
+          <meta property="og:image" content="https://webpack.js.org/icon_180x180.png" />
```

Comment:
```suggestion
          <meta property="og:image" content={ `https://webpack.js.org${Logo}` } />
```

Reply:
@EugeneHlushko 
Thank you for review.
IMHO,
I think that `icon_180x180.png` is better than `assets/logo-on-white-bg.svg` at Twitter card.

https://webpack.js.org/icon_180x180.png

https://cards-dev.twitter.com/validator

Reply:
Thank you for your honest opinion, although we dont link to assets directly, we use hashed assets (from live website):
```
<meta property="og:image" content="/aad9d51e01a0dcc2caf5bee43dc8e647.svg"/>
```

Add the domain and it works

Reply:
@EugeneHlushko 
Thank you for reply.
I selected your suggestion.